### PR TITLE
feat(onyx-705): viewing rooms notification page

### PR DESCRIPTION
**Pull request by nickskalkin**
The type of this PR is: **Feat**

This PR solves [ONYX-705]

### Description

Adds new viewing room notification page. There are no designs for this page - I've made it based on my assumptions, but it's better then nothing. I'll ping Barney to review and give feedback.

Demo:

<video src="https://github.com/artsy/force/assets/3934579/cb29c381-65f4-4b7b-b345-f8de87e35fee" />

[ONYX-705]: https://artsyproduct.atlassian.net/browse/ONYX-705?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by nickskalkin** [comment]
@tam-kis @dariakoko just talked to Barney and he approved the design

**Review by damassi — COMMENTED** [review]
This looks good! Would be nice to add a few tests before merging, but otherwise 👍

File: src/Components/Notifications/NotificationViewingRoom.tsx [hunk 1]
```diff
@@ -0,0 +1,106 @@
+import * as React from "react"
+import { graphql, useFragment } from "react-relay"
+import { RouterLink, RouterLinkProps } from "System/Router/RouterLink"
+import { NotificationViewingRoom_viewingRoom$key } from "__generated__/NotificationViewingRoom_viewingRoom.graphql"
+import { AuthContextModule } from "@artsy/cohesion"
+import { Box, Button, Image, Spacer, Text } from "@artsy/palette"
+import { resized } from "Utils/resized"
+
+const MAX_WIDTH = 600
```
Comment (dariakoko) [inline src/Components/Notifications/NotificationViewingRoom.tsx:9]: If we are reusing this value, do you think we could move it to one of the parent components?

**Review by dariakoko — DISMISSED** [review]
Looks very good!
Only left one not blocking comment
And agree with Devon, let's add some tests or add a ticket to add tests

**Review by anandaroop — DISMISSED** [review]
Nice

**Comment by nickskalkin** [comment]
thanks @damassi, added tests

File: src/Components/Notifications/NotificationViewingRoom.tsx (hunk 1 above)
Comment (nickskalkin) [reply src/Components/Notifications/NotificationViewingRoom.tsx:9]: @dariakoko yes, I already did in a separate PR here: https://github.com/artsy/force/pull/13456/files#diff-edd4e49eedcd30401192b266b9596e84735f5fbf6bb8f30ca09871a0bbbc9fa7R1

**Pull request merged**